Add Recaptcha Verification

fatlotus opened the pull request:
Test URL: https://add-recaptcha-dot-caravel-code-reviews.appspot.com.

I've rushed this into prod, but I'd still like approval before merging into master.

georgeteo commented on caravel/__init__.py:
```diff
@@ -7,15 +7,20 @@
-from flask import Flask
-app = Flask(__name__)
+app.config["RECAPTCHA_PUBLIC_KEY"] = "6LeeKBITAAAAAPVBZTwJPsfBBIwp68QKG5tMQOz4"
+app.config["RECAPTCHA_PRIVATE_KEY"] = "6LeeKBITAAAAAEW1o535C9A7AE2dNauMcay5_2Jr"
```
Do we want the private key visible to anybody who looks at the code?

georgeteo commented:
Looks good except for the privkey concern above.

fatlotus commented:
Whoops! Deleted it and moved it into config, but I guess I forgot the one in init. In any case I've killed the key and generated a new one and will update the commit shortly.

fatlotus commented:
Updated!
